userWhitelist.json

```diff
@@ -14,18 +14,16 @@
     "chris7716"
   ],
   "goodFirstIssue": [
-    "U8NWXD",
     "kevintab95",
+    "FaithAbiola",
+    "jayam04",
+    "ojasvi004",
```

Comment:
Please take ojasvi004 off, she is not a lead of the team currently. (I just updated the team tracker, sorry about that!)